admin_app/src/app/content/edit/page.tsx

```diff
@@ -138,6 +199,11 @@ const ContentBox = ({
       language_id: content.language_id,
       content_metadata: content.content_metadata,
     };
+    if (body.language_id === 0) {
+      setErrorText("Please select a language");
```

Comment:
I am assuming you think it shouldn't do that because if you are adding a new language and then noticed it is the wrong language, everything will deleted. But I did that   because I noticed that if I don't, when you go from an already existing language version to a non existing language version, then you have the content of the old language and it is quite annoying to have to delete everything manually before you start typing. A tradeoff would be: When we go from a saved language version to a non existing language the content is deleted. Then if we add values to the text and title, the fields are not emptied until we click on an existing language version in the DB. What do you think ? This is what is currently hapening with the changes I made. But I am happy to update that if you think always keeping the previous content is better.

Reply:
I see.. I think from user's perspective a smoother experience might be

1. Add new language -- app changes state to _editing_ or something
2. Add title & text
3. Click on English (without clicking Save) -- app asks if you want to save changes. 
4. Click on Save
5. Then you are able to click on English

If this doesn't seem like a 10min job, let's create a backlog ticket.

Reply:
Aha. Relatedly, when I'm on Add Content page,

1. POST /language to create new language (e.g. HINDI)
1. Without filling out title or body, click on language ➕  and click Hindi
  - This removes the English tab (bc it was empty, I guess?) . 
3. Now enter some title and body in Hindi tab
4. Click language ➕ button and click English
5. **Now English tab is filled with same content as 3.**


I think this deserves a more thorough thinking around UI/UX. cc: @Tanmay-97

Reply:
Another reason why I think we need to thoroughly consider better UI/UX is because of the delete button's functionality under each language tab.

Right now, 
1. when it's an unsaved language version, delete button just deletes the language tab.
2. If it's a saved language version with no other language version, the same button can delete the entire content. 

We need some visual distinction.. For example, don't allow deleting last remaining  language version in an edit page. And/or have a separate content-level delete button outside the language tabs.

<img width="1113" alt="image" src="https://github.com/IDinsight/aaq-core/assets/7042047/23a343b0-4eeb-4c7b-a839-726cf4ceff46">

Also, it seems too powerful to be able to delete an entire content from just content card landing page?! But that's up for debate

Reply:
I think this is something thats going to be hard to discuss by text let's talk about it during our checkin. Then I can make the changes later